tests/TestCase/Utility/SecurityTest.php

```diff
@@ -70,6 +70,20 @@ public function testHash()
     }
 
     /**
+     * testInvalidHashTypeException
+     *
+     * @return void
+     */
+    public function testInvalidHashTypeException() {
+        try {
+            Security::hash('test', 'doesnotexist', false);
+            $this->fail('Expected \RuntimeException');
+        } catch (\RuntimeException $e) {
+            $this->assertTextContains('The hash type `doesnotexist` was not found. Available algorithms are:', $e->getMessage());
```

Comment:
s/type/algo

Reply:
Can it be achieved using `@expectedException` and `@expectedExceptionMessage` annotations?

Reply:
@robertpustulka can you do a regex or a partial match for the assert of the message?

Reply:
@burzum yes, you can:

https://phpunit.de/manual/current/en/appendixes.annotations.html#appendixes.annotations.expectedExceptionMessageRegExp